app/gunicorn.conf.py

```diff
@@ -0,0 +1,21 @@
+"""
+Configuration file for the Gunicorn server used to run the application in production environments.
+
+Attributes:
+    bind(str): The socket to bind. Formatted as '0.0.0.0:$PORT'.
+    threads(int): The number of threads per worker for handling requests.
+
+For more information, see https://docs.gunicorn.org/en/stable/configure.html
+"""
+
+from src.app_config import AppConfig
+
+app_config = AppConfig()
+
+bind = app_config.host + ':' + str(app_config.port)
+# Calculates the number of usable cores and doubles it. Recommended number of workers per core is two.
```

Comment:
Where is the number of workers set?
https://github.com/navapbc/template-application-flask/blob/main/app/gunicorn.conf.py and https://docs.gunicorn.org/en/stable/configure.html show setting the workers explicitly.

Reply:
By [default](https://docs.gunicorn.org/en/stable/settings.html#workers) the number of worker values is 1. When I set the workers to 2+. I ended up with this error:
```
/Users/charlie/projects/dst/labs-decision-support-tool/app/.venv/lib/python3.12/site-packages/threadpoolctl.py:1214: RuntimeWarning: 
Found Intel OpenMP ('libiomp') and LLVM OpenMP ('libomp') loaded at
the same time. Both libraries are known to be incompatible and this
can cause random crashes or deadlocks on Linux when loaded in the
same Python program.
Using threadpoolctl may cause crashes or deadlocks. For more
information and possible workarounds, please see
    https://github.com/joblib/threadpoolctl/blob/master/multiple_openmp.md

  warnings.warn(msg, RuntimeWarning)
/Users/charlie/projects/dst/labs-decision-support-tool/app/.venv/lib/python3.12/site-packages/threadpoolctl.py:1214: RuntimeWarning: 
Found Intel OpenMP ('libiomp') and LLVM OpenMP ('libomp') loaded at
the same time. Both libraries are known to be incompatible and this
can cause random crashes or deadlocks on Linux when loaded in the
same Python program.
Using threadpoolctl may cause crashes or deadlocks. For more
information and possible workarounds, please see
    https://github.com/joblib/threadpoolctl/blob/master/multiple_openmp.md
```
I followed the work arounds in the site but those didn't seem to fix the issue for me, using the default 1 worker did.

Reply:
Setting workers = 4 works for me. Do you also get that error when you run it in Docker?

Reply:
Running it on Docker seems to work, I was testing with the command `poetry  run  gunicorn -w 4 -k uvicorn.workers.UvicornWorker -b 0.0.0.0:8000  src.app:app`